tests/backends/test_arviz.py

```diff
@@ -454,7 +454,7 @@ def test_constant_data(self, use_context):
         test_dict = {
             "posterior": ["beta"],
             "observed_data": ["obs"],
-            "constant_data": ["x", "y", "beta_sigma"],
+            "constant_data": ["x", "beta_sigma"],
```

Comment:
This `y` was in fact observed data... Now nothing stops the user from using `y` both as observed data and constant data, so I am not sure we want to truly remove it.

We can check if `y` shows up in the generative graph.

Reply:
doing it now